app/components/Code/CodeBlock.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+import { useEffect } from "react";
+import hljs from "highlight.js";
+
+import { Icon } from "~icon/index";
+
+type CodeBlockProps = {
+  code: string;
+  className?: string;
+  language?: string;
+  fileName?: string;
+};
+
+const CodeBlock = ({
+  className,
+  code,
+  language = "",
+  fileName,
+}: CodeBlockProps) => {
+  useEffect(() => {
+    hljs.highlightAll();
+  }, []);
```

Comment:
`yarn build` doesn't fail, but when you go to `/work/webflow` it throws a client side exception.
![Screenshot 2024-02-09 at 1 48 29 AM](https://github.com/sarmstead/mi-oficina/assets/65554107/73b45eb5-8bf7-4194-b4fd-ac1d167201f1)